### PR TITLE
docs: Add JSDocs for exported methods and types

### PR DESCRIPTION
**Pull request by KnightNiwrem**
Based off https://github.com/grammyjs/parse-mode/pull/12

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -21,12 +21,12 @@
...
-    "@grammyjs/types": "^2.9.1",
+    "@grammyjs/types": "^2.11.0",
     "@tsconfig/node16": "^1.0.2",
     "@types/node": "^16.6.1",
-    "deno2node": "^1.4.0",
-    "grammy": "^1.11.2",
-    "typescript": "^4.5.5"
+    "deno2node": "^1.7.1",
+    "grammy": "^1.13.0",
+    "typescript": "^4.9.4"
```
Comment (KnorpelSenf) [inline package.json:29]: What is TS used for?

File: src/format.ts [hunk 2]
```diff
@@ -46,21 +64,67 @@ const buildFormatter = <T extends Array<any> = never>(
   };
 };
 
-// Native entity functions
+// === Native entity functions
```
Comment (KnorpelSenf) [inline src/format.ts:67]: You stole my `// ===` syntax, and I appreciate it :D

File: src/format.ts [hunk 3]
```diff
@@ -1,13 +1,31 @@
 import type { MessageEntity } from "./deps.deno.ts";
 
+/**
+ * Objects that implement this interface implements a `.toString()` 
+ * method that returns a `string` value that represents the object.
+ */
 export interface Stringable {
   toString(): string;
```
Comment (KnorpelSenf) [inline src/format.ts:8]: It probably make sense to document these properties, too

File: src/format.ts [hunk 4]
```diff
@@ -1,13 +1,31 @@
 import type { MessageEntity } from "./deps.deno.ts";
 
+/**
+ * Objects that implement this interface implements a `.toString()`
```
Comment (roziscoding) [inline src/format.ts:4]:
```suggestion
 * Objects that implement this interface implement a `.toString()` 
```

File: src/format.ts [hunk 5]
```diff
@@ -1,13 +1,31 @@
 import type { MessageEntity } from "./deps.deno.ts";
 
+/**
+ * Objects that implement this interface implements a `.toString()` 
+ * method that returns a `string` value that represents the object.
```
Comment (roziscoding) [inline src/format.ts:5]:
```suggestion
 * method that returns a `string` value representing the object.
```

IMO repeating "that" makes it a. bit confusing, specially for non-native english speakers :)

**Review by roziscoding — APPROVED** [review]
Other than the proposed changes, LGTM :)

**Review by KnorpelSenf — CHANGES_REQUESTED** [review]
Looks pretty good to me! Thank you so much!

Just a few comments:

File: package.json (hunk 1 above)
Comment (KnightNiwrem) [reply package.json:29]: For filling up the disk space of contributors!

Comment (KnorpelSenf) [reply package.json:29]: Great idea!
```suggestion
    "typescript": "^4.9.4",
    "lodash": ">=1.0.0"
```

Comment (KnorpelSenf) [reply package.json:29]: Uhm so you're going to remove it?

Comment (KnightNiwrem) [reply package.json:29]: *sweating profusely, and looking around nervously*

(Yes)

Comment (KnorpelSenf) [reply package.json:29]: Haha nice :D

There's no hurry, I just wasn't sure about the plan lol

**Review by KnorpelSenf — APPROVED** [review]
LGTM!

**Comment by KnorpelSenf** [comment]
Nice branch name btw lol

**Pull request merged**